Framework/Subset_of_CorLib/System/DateTime.cs

```diff
@@ -93,13 +88,18 @@ public struct DateTime
         private const ulong TickMask = 0x7FFFFFFFFFFFFFFFL;
         private const ulong UTCMask = 0x8000000000000000L;
 
-        public static readonly DateTime MinValue = new DateTime(MinTicks);
-        public static readonly DateTime MaxValue = new DateTime(MaxTicks);
+        // Ticks at 12:00 AM January 1, year 1601 A.D.

```

Comment:
So what is the rationale behind carrying this workaround in managed code and not fixing it in native? What code could potentially break in existing HAL?